app/uk/gov/hmrc/plasticpackagingtax/registration/config/AppConfig.scala

```diff
@@ -73,4 +76,11 @@ class AppConfig @Inject() (config: Configuration, val servicesConfig: ServicesCo
   def incorpDetailsUrl(journeyId: String): String = s"$incorpJourneyUrl/$journeyId"
 
   def pptRegistrationUrl(id: String): String = s"$pptRegistrationUrl/$id"
+
+  def authenticatedFeedbackUrl(): String =
+    s"$contactFrontendHost/contact/beta-feedback?service=${serviceIdentifier}"
+
+  def unauthenticatedFeedbackUrl(): String =
+    s"$contactFrontendHost/contact/beta-feedback-unauthenticated?service=${serviceIdentifier}"
```

Comment:
Do we have an agreed format we stick to in general for these types of URLs?

Reply:
These URLs are from third party microservices